src/Exception/InvalidArgumentException.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the fXmlRpc package.
+ *
+ * (c) Lars Strojny <lstrojny@php.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace fXmlRpc\Exception;
+
+use fXmlRpc\Exception;
+
+/**
+ * @author Lars Strojny <lstrojny@php.net>
+ */
+final class InvalidArgumentException extends \InvalidArgumentException implements Exception
```

Comment:
Isn’t that an implementation detail and should be moved outside so we just keep the Exception interface?